src/api/app/models/bs_request_action.rb

```diff
@@ -74,7 +74,7 @@ def self.get_package_diff(path, query)
   end
 
   def self.type_to_class_name(type_name)
-    "bs_request_action_#{type_name}".classify.constantize
+    "BsRequestAction#{type_name.classify}"
```

Comment:
this can not work. `constantize` tries to find a declared constant with given name, that is a `Class` or `Module` for example. Now you are returning a String 🙈

Reply:
ouch, let me change it